src/Microsoft.ML.Data/DataLoadSave/EstimatorChain.cs

```diff
@@ -15,7 +15,7 @@ namespace Microsoft.ML.Data
     public sealed class EstimatorChain<TLastTransformer> : IEstimator<TransformerChain<TLastTransformer>>
         where TLastTransformer : class, ITransformer
     {
-        // Host is not null iff there is any 'true' values in _needCacheAfter (in this case, we need to create an instance of
+        // Host is not null if there are any 'true' values in _needCacheAfter (in this case, we need to create an instance of
```

Comment:
`iff` is not a type-o. It is intended. See "Entry 1" on https://www.merriam-webster.com/dictionary/iff.

Reply:
OK, thanks. Have reverted and also correct other small typo in the same sentence. `is` should be `are` though I believe.